src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -2494,6 +2506,24 @@ private void DecodeWellKnownAttribute(ref DecodeWellKnownAttributeArguments<Attr
             else if (attribute.IsTargetAttribute(this, AttributeDescription.GuidAttribute))
             {
                 attribute.DecodeGuidAttribute(arguments.AttributeSyntaxOpt, diagnostics);
+                if (attribute.TryGetGuidAttributeValue(out string guidString))
```

Comment:
>TryGetGuidAttributeValue [](http://example.com/codeflow?start=30&length=24)

It loos like ```DecodeGuidAttribute``` already returns the string

Reply:
But I now realize `[Guid(null)]` was accepted previously and isn't now, so there's a behavior change anyway.